src/test/java/entities/PostTest.java

```diff
@@ -114,15 +114,15 @@ public void getDeadline() {
     public void getReply(){
         LocalDate deadline = LocalDate.of(2018, 12, 31);
         Post newPost = new Post(poster, title, mainDesc, tags, collaborators, deadline, 0);
-        Comment comment = new Comment(poster, "Hello!");
+        Comment comment = new Comment(poster, "Hello!", 0);
```

Comment:
This looks like it's still using the old entities. For example, `Comment` no longer accepts a `User` object as a parameter. I think Johann's PR fixes this so you can remove this file from the commits.

Reply:
I think this one was accidentally committed. I'll remove it